packages/frontend/src/main/lib/viewer/commit-object-viewer/stateManager.ts

```diff
@@ -40,28 +45,33 @@ let globalViewerData: Nullable<GlobalViewerData> = null
  */
 const commitObjectViewerState = makeVar({
   viewerBusy: false,
-  appliedFilter: null as Nullable<UnknownObject>,
-  isolateKey: null as Nullable<string>,
-  isolateValues: [] as string[],
-  hideKey: null as Nullable<string>,
-  hideValues: [] as string[],
-  colorLegend: {} as UnknownObject,
-  isolateCategoryKey: null as Nullable<string>,
-  isolateCategoryValues: [] as string[],
-  hideCategoryKey: null as Nullable<string>,
-  hideCategoryValues: [] as string[],
   selectedCommentMetaData: null as Nullable<{
     id: number
     selectionLocation: Record<string, unknown>
   }>,
   addingComment: false,
   preventCommentCollapse: false,
   commentReactions: ['❤️', '✏️', '🔥', '⚠️'],
-  emojis
+  emojis,
+  // New viewer & filter vars
+  currentFilterState: null as Nullable<FilteringState>,
+  selectedObjects: [] as UnknownObject[],
+  objectProperties: [] as PropertyInfo[],
```

Comment:
remember to update local-only/schema.gql with these as well, otherwise we wont get proper types in queries